src/ArticleCard.jsx

```diff
@@ -48,9 +49,10 @@ function ArticleCard() {
       <p>{article.body}</p>
       <p>By {article.author}</p>
       <p>Posted on: {article.created_at}</p>
+      <p>Votes: {article.votes}</p>
       <p>Votes: {voteCount}</p>
```

Comment:
Why is there 2 votes shown on the page, sometimes these are different and can be clicked multiple times, please only allow a vote in either direction of 1 per refresh. Unvote would also be nice but not essential